Packs/Phishing/Scripts/PhishingDedupPreprocessingRule/PhishingDedupPreprocessingRule.py

```diff
@@ -47,16 +50,22 @@ def get_existing_incidents(input_args):
     elif 'exsitingIncidentsLookback' in DEFAULT_ARGS:
         get_incidents_args['fromDate'] = DEFAULT_ARGS['exsitingIncidentsLookback']
     status_scope = input_args.get('statusScope', 'All')
+    query_components = []
+    if 'query' in get_incidents_args:
+        query_components.append(get_incidents_args['query'])
     if status_scope == 'ClosedOnly':
-        if 'query' in get_incidents_args:
-            get_incidents_args['query'] = '({}) and (status:Closed)'.format(get_incidents_args['query'])
-        else:
-            get_incidents_args['query'] = 'status:Closed'
+        query_components.append('status:Closed')
     elif status_scope == 'NonClosedOnly':
-        if 'query' in get_incidents_args:
-            get_incidents_args['query'] = '({}) and (-status:Closed)'.format(get_incidents_args['query'])
-        else:
-            get_incidents_args['query'] = '-status:Closed'
+        query_components.append('-status:Closed')
+    elif status_scope == 'All':
+        pass
+    else:
+        return_error('Unsupported statusScope: {}'.format(status_scope))
+    incident_type = input_args.get('incidentTypes', DEFAULT_ARGS['incidentTypes'])
```

Comment:
I don't think we need to force incident type,  it it's empty just skip this part in the query